src/Checkout/EventSubscriber/CheckoutEventSubscriber.php

```diff
@@ -43,30 +46,27 @@
 
 class CheckoutEventSubscriber implements EventSubscriberInterface
 {
-    /**
-     * @var Ps_checkout
-     */
-    private $module;
-
-    /**
-     * @var CommandBusInterface
-     */
-    private $commandBus;
-
-    /**
-     * @var CheckoutChecker
-     */
-    private $checkoutChecker;
-    /**
-     * @var PsCheckoutCartRepository
-     */
-    private $psCheckoutCartRepository;
-
-    public function __construct(CheckoutChecker $checkoutChecker, CommandBusInterface $commandBus, PsCheckoutCartRepository $psCheckoutCartRepository)
-    {
+    private QueryBusInterface $queryBus;
+    private CheckoutChecker $checkoutChecker;
+    private PsCheckoutCartRepository $psCheckoutCartRepository;
+    private UpdatePaymentMethodSelectedCommandHandler $updatePaymentMethodSelectedCommandHandler;
+    private CreateOrderCommandHandler $createOrderCommandHandler;
+    private CapturePayPalOrderCommandHandler $capturePayPalOrderCommandHandler;
+
+    public function __construct(
+        CheckoutChecker $checkoutChecker,
+        PsCheckoutCartRepository $psCheckoutCartRepository,
+        QueryBusInterface $queryBus,
+        UpdatePaymentMethodSelectedCommandHandler $updatePaymentMethodSelectedCommandHandler,
+        CreateOrderCommandHandler $createOrderCommandHandler,
+        CapturePayPalOrderCommandHandler $capturePayPalOrderCommandHandler
```

Comment:
```suggestion
        private readonly CheckoutChecker $checkoutChecker,
        private readonly PsCheckoutCartRepository $psCheckoutCartRepository,
        private readonly QueryBusInterface $queryBus,
        private readonly UpdatePaymentMethodSelectedCommandHandler $updatePaymentMethodSelectedCommandHandler,
        private readonly CreateOrderCommandHandler $createOrderCommandHandler,
        private readonly CapturePayPalOrderCommandHandler $capturePayPalOrderCommandHandler
```
Little tip since you now use PHP 8.1 minimum, you can use the property promotion feature, if you want, that greatly decreases the quantity of code https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion